sdk/sdk.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Mia srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sdk
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rond-authz/rond/core"
+	"github.com/rond-authz/rond/internal/metrics"
+	"github.com/rond-authz/rond/openapi"
+
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
we still have prometheus here (and logrus but that's for another PR for sure) are we still planning to remove it from SDK, right?